Firestore/Source/Model/FSTMutation.h

```diff
@@ -197,6 +199,7 @@ typedef NS_ENUM(NSUInteger, FSTPreconditionExists) {
  * FSTSetMutation, but not necessarily for an FSTPatchMutation).
  */
 - (FSTMaybeDocument *_Nullable)applyTo:(FSTMaybeDocument *_Nullable)maybeDoc
```

Comment:
Can't these all be `nullable FSTMaybeDocument *`?

Reply:
Yes. I'm still mightily confused on what to use where, so I usually go with examples from the current source file.